day-13/index.js

```diff
@@ -0,0 +1,201 @@
+
+        let sum1 = 0;
+        let quantity2 = 0;
+        array.filter(x => isInteger(x)).forEach(x => {
+            sum1 += x;
+            quantity2++;
+        });
+        if (quantity2 > 0) {
+            console.log(`e) the arithmetic average = ${sum1 / quantity2}`);
+        }
+
+        let sum2 = 0;
+        let quantity3 = 0;
+        array.filter(x => isInteger(x)).forEach(x => {
+            if (x < 0 && x % 7 === 0) {
+                sum2 += x;
+                quantity3++;
+            }
+        });
+
+        if (quantity3 > 0) {
+            console.log(`f) the arithmetic average of negative = ${sum2 / quantity3}`);
+        }
+
+        let array1 = array.filter(x => isInteger(x));
+        let quantity4 = 0;
+        for (let i = 0; i < array1.length; i++) {
+            let sliceFirstIndex = Number(array1[i].toString().replaceAll("-", "").slice(0, 1));
+            if (sliceFirstIndex === 3) {
+                quantity4++;
+            }
+        }
+        console.log(`g) quantity of first digit begins 3 = ${quantity4}`);
+
+        let array2 = array.filter(x => isInteger(x) && x > 0 && x % 2 === 0);
+        let array3 = array.filter(x => isInteger(x) && x < 0);
+        let concatArray = array2.concat(array3);
+        console.log(`h) quantity of even and positive numbers and then negative numbers = ${concatArray}`);
+    }
+
+    integerNumbers([1, 2.5, 0, -7, -3, 8, 4.6, 30, 9, -34, 6, -5, 13, 314]);
+}
+
+
+/** 2. Տրված  է  իրական  թվերի  հաջորդականություն: Արտածել  այդ
+ հաջորդականության
+ ա) այն տարրերի քանակը, որոնք փոքր են իրենց կարգահա-
+ մարի քառակուսուց,
+ բ) բոլոր  տարրերի  միջին  թվաբանականի  եւ  տարրերի  տար-
+ բերությունները,
+ գ) այն  տարրերի  քանակը,  որոնք  փոքր  են  իրենց  ձախ  եւ  աջ
+ հարեւանների կիսագումարից,
+ դ) մեծագույն տարրը եւ նրա կարգահամարը,
+ ե) մեծագույն եւ փոքրագույն տարրերի տարբերությունը,
+ զ) առաջին մեծագույն եւ վերջին փոքրագույն տարրերի միջեւ
+ գտնվող անդամների  գումարը,
+ է) մեծագույն եւ մեծությամբ երկրորդ տարրերը,
+ ը) դրական  տարրերից  փոքրագույնի  արժեքը  ( եթե  դրական
+ տարր  չկա,  արտածել  համապատասխան  հաղորդագրու-
+ թյուն): */
+
+async function problem3() {
+    function realNumbersArray(array) {
+        console.log(`${parb('input array is:')} ${parg(array)}`);
+
+        let quantity = 0;
+
+        for (let i = 0; i < array.length; i++) {
+            if (array[i] < i ** 2) {
+                quantity++;
+            }
+        }
+        console.log(`a) the number of elements that are small in their index square = ${quantity}`);
+
+
+        let sum = 0;
+        let quantity1 = 0;
+        let arithmeticAverage = sum / quantity1;
+        let difference = "";
+
+        for (let i = 0; i < array.length; i++) {
+            sum += array[i];
+            quantity1++;
+            arithmeticAverage = sum / quantity1;
+        }
+
+        for (let i = 0; i < array.length; i++) {
+
+            if (arithmeticAverage > array[i]) {
```

Comment:
it asks you to find the difference between `arithmeticAverage` and array elements but does not restrict to find only 
positive difference